api/v2/serializers/fields/identity.py

```diff
@@ -7,9 +7,7 @@ class IdentityRelatedField(serializers.RelatedField):
     def get_queryset(self):
         return Identity.objects.all()
 
-    def to_representation(self, value):
-        queryset = self.get_queryset()
-        identity = queryset.get(pk=value.pk)
```

Comment:
@steve-gregory, I'm guessing this change is going to break other code. Need to find out what is depending on this behavior.